src/Plugin/Field/FieldType/OgStandardReferenceItem.php

```diff
@@ -3,50 +3,22 @@
 namespace Drupal\og\Plugin\Field\FieldType;
 
 use Drupal\Core\Field\Plugin\Field\FieldType\EntityReferenceItem;
-use Drupal\Core\Form\FormStateInterface;
 
 /**
  * Class OgStandardReferenceItem.
  *
  * @FieldType(
  *   id = "og_standard_reference",
- *   label = @Translation("OG membership reference"),
- *   description = @Translation("An entity field containing an OG membership reference for a non-user entity."),
+ *   label = @Translation("OG reference"),
+ *   description = @Translation("An entity field containing an OG reference for a non-user entity."),
```

Comment:
It extends EntityReferenceItem so ... "An entity reference field" ...
```suggestion
 *   description = @Translation("An entity reference field containing an OG reference for a non-user entity."),
```

Reply:
What is meant with non-user entity? Isn't this a reference to the group (which could be any type of entity)?